app/workers/evss/disability_compensation_form/submit_form526.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+module EVSS
+  module DisabilityCompensationForm
+    class SubmitForm526
+      include Sidekiq::Worker
+
+      FORM_ID = '21-526EZ'
+
+      # Sidekiq has built in exponential back-off functionality for retrys
+      # A max retry attempt of 10 will result in a run time of ~8 hours
+      RETRY = 10
+
+      sidekiq_options retry: RETRY
+
+      # This callback cannot be tested due to the limitations of `Sidekiq::Testing.fake!`
+      sidekiq_retries_exhausted do |_msg, _ex|
+        transaction_class.update_transaction(jid, :exhausted)
+      end
+
+      # Performs an asynchronous job for submitting a form526 to an upstream
+      # submission service (currently EVSS)
+      #
+      # @param user_uuid [String] The user's uuid thats associated with the form
+      # @param form_content [Hash] The form content that is to be submitted
+      # @param uploads [Hash] The users ancillary uploads that will be submitted separately
+      #
+      def perform(user_uuid, form_content, uploads)
+        user = User.find(user_uuid)
+
+        transaction_class.start(user, jid) if transaction_class.find_transaction(jid).blank?
+        response = service(user).submit_form(form_content)
+        transaction_class.update_transaction(jid, :received, response.attributes)
+
+        Rails.logger.info('Form526 Submission',
+                          'user_uuid' => user.uuid,
+                          'job_id' => jid,
+                          'job_status' => 'received')
+
+        # Delete SiP form on successfull submission
```

Comment:
What happens in the unlikely event the clean up code fails? Is it rescued by the StandardError and recorded as a `:non_retryable_error` (even though submit went through)? Maybe clean up could be a separate job? That way the clean up job could still retry its commands.